ouroboros-network-testing/src/Ouroboros/Network/Testing/Utils.hs

```diff
@@ -20,6 +20,13 @@ newtype Delay = Delay { getDelay :: DiffTime }
   deriving newtype (Eq, Ord, Num)
 
 
+genDelayWithPrecision :: Integer -> Gen DiffTime
+genDelayWithPrecision precision = 
+    sized $ \n -> do
+      b <- chooseInteger (1, precision)
+      a <- chooseInteger (0, toInteger n * b)
+      return (fromRational (a % b))
+
```

Comment:
I think you can remove this commit, the `genDelayWithPrecision` is included below. Probably an artefact of a rebase.